### ISSUE TITLE
Transform not being called on test set

### ISSUE THREAD

**Issue by fayalalebrun** [V0XNIHILI/few-shot-spoken-term-classification#2]
`test_data = SPEECHCOMMANDS_12C(root, url, folder_in_archive, download=download, subset='testing', target_transform=test_target_transform)`
Should be:
`test_data = SPEECHCOMMANDS_12C(root, url, folder_in_archive, download=download, subset='testing', transform=transform, target_transform=test_target_transform)`

**Comment by fayalalebrun** [comment]
Unless this is done one purpose in order to do some transforms specifically on the train set only?

I am using the transform in order to transpose the data columns.

**Comment by V0XNIHILI** [comment]
Yep thats done on purpose as to not augment the test set. Should I add a custom test_transform parameter?

**Comment by fayalalebrun** [comment]
It can be worked around with the dataloader collate_fn. However some documentation pointing this out may be useful for others to not repeat the same mistake.

**Comment by V0XNIHILI** [comment]
Fixed with latest commit (8fe5d2c)!
